docs: Reshape Endpoints & APIs concepts, archive deprecated streaming (#673)
serverpod/serverpod_docs: docs/06-concepts/01-working-with-endpoints/01-working-with-endpoints.md

---
slug: /concepts/working-with-endpoints
sidebar_label: Working with endpoints

> description: Define server methods as endpoints, generate a typed client, and call them from your Flutter app, including pointing the client at each environment.

Reviewer: Non-blocking: `description` reads better leading with the topic noun rather than a verb (per Alex's guidance on #635).

```suggestion
description: Endpoints expose server methods to a generated, typed client your Flutter app calls, with one client setup pointed at each environment.
```
